book/src/building/swift-package/README.md

```diff
@@ -0,0 +1,246 @@
+# Bundling rust code as a Swift package
```

Comment:
Need to add this file to `book/src/SUMMARY.md`

Also should capitalize `Rust` across the file.